main.py

```diff
@@ -1,47 +1,66 @@
 import sys
 
+def command_failed():
+    return None
 
 def get_arguments(arguments):
     arguments.pop(0)
 
     number_of_arguments = len(arguments)
     print("Number of arguments", number_of_arguments)
 
-    if number_of_arguments <= 1 or number_of_arguments > 2:
+    if number_of_arguments != 2:
```

Comment:
Should be` if number_of_arguments < 1 or number_of_arguments > 2:` , since the 2nd arg (max crawl) is optional

Reply:
Commit [1a27893] adds this in